### ISSUE TITLE
Doesn't start on 23H2 (22631.2428)?

### ISSUE THREAD

**Issue by eggbean** [hwtnb/SylphyHornPlusWin11#36]
I don't use the Insider Builds to avoid problems like this, but I have just upgraded to 23H2 official mainline update and now `SylphyHornPlusWin11` isn't even starting 😭

```
Edition	Windows 11 Pro
Version	23H2
Installed on	‎23/‎07/‎2023
OS build	22631.2428
Experience	Windows Feature Experience Pack 1000.22674.1000.0
```

**Comment by moridonut** [comment]
It starts in my case, but it is shut down as soon as I switch to the next desktop.
I wonder if there is a way to disable this popup by vanilla Windows.
![image](https://github.com/hwtnb/SylphyHornPlusWin11/assets/12019065/376f2453-c47a-440b-b52f-387f68e8f284)

**Comment by eggbean** [comment]
There doesn't seem to be anything in the Settings. If it is possible it would probably be something like a registry setting. Anyway, I don't mind that. It's nicer than the SlyphyHorn one, which I don't use.

I've only just noticed that now there is now scrolling animation when using the keyboard to change desktop. Before this update the change was immediate and the scrolling animation only worked when using trackpad gestures to change desktop. This change is probably what has broken SlyphyHornPlus11.

**Comment by w3b-beweb** [comment]
I can confirm that on Win11 Pro 23H2 (build 22631.2506), SylphyHornPlusWin11 starts but crashes as soon as I change to next/prev virtual desktop or use my shortcuts to jump to a certain virtual desktop. It crashes before showing the desktop name label.

**Comment by drothenberger** [comment]
> I can confirm that on Win11 Pro 23H2 (build 22631.2506), SylphyHornPlusWin11 starts but crashes as soon as I change to next/prev virtual desktop or use my shortcuts to jump to a certain virtual desktop. It crashes before showing the desktop name label.

It's working for me with Win11 Pro on the same build. It works both with SH notifications enabled and disabled. I did have the crash occasionally, but toggling the SH notifications solved the problem.

**Comment by w3b-beweb** [comment]
> It's working for me with Win11 Pro on the same build. It works both with SH notifications enabled and disabled. I did have the crash occasionally, but toggling the SH notifications solved the problem.

I can't get it to work. Launching it as administrator, toggling notification, resetting preferences... no way. It crashes every time at each desktop switch. Looking In windows app event viewer, i've found these two error messages for each crash:

```
Application: SylphyHorn.exe
Framework version: v4.0.30319
Description: The process was terminated due to an internal .NET runtime error 
at IP 00007FFC19C6131E (00007FFC19AD0000) with exit code 80131506. 
```
and

```
Name of the application that generated the error: SylphyHorn.exe, version: 4.0.0.0, timestamp: 0xd3f9f535
Name of the module that generated the error: clr.dll, version: 4.8.9181.0, timestamp: 0x64b85478
Exception code: 0xc0000005
Offset error 0x000000000019131e
Process ID that generated the error: 0x0x1BAC
Launch time of the application that generated the error: 0x0x1DA13A1949774A5
Path to the application that generated the error: C:\Program Files (portable)\SylphyHorn\SylphyHorn.exe
Path to the module that generated the error: C:\Windows\Microsoft.NET\Framework64\v4.0.30319\clr.dll
```

I can provide the app crash dump file if needed

**Comment by drothenberger** [comment]
Have you tried the troubleshooting step mentioned in the release notes?

Troubleshooting for update issues (beta.10 or earlier):

It is recommended to delete cache data (VirtualDesktop.*****.generated.dll) at %UserProfile%\AppData\Local\hwtnb.net\SylphyHorn\assemblies if error occurs.

**Comment by w3b-beweb** [comment]
> Have you tried the troubleshooting step mentioned in the release notes?
> 
> Troubleshooting for update issues (beta.10 or earlier):
> 
> It is recommended to delete cache data (VirtualDesktop.*****.generated.dll) at %UserProfile%\AppData\Local\hwtnb.net\SylphyHorn\assemblies if error occurs.

OMG:anguished: Totally overlooked that... as a developer, I'm disappointed in myself for this :disappointed::unamused:
thank you so much!

**Comment by eggbean** [comment]
> Have you tried the troubleshooting step mentioned in the release notes?

I didn't think of doing that as it was Windows that got updated and not SylphyHornPlusWin11. I deleted the two files I had in `assemblies` but it didn't help. It's still not working for me.

@w3b-beweb So it's working for you now?

**Comment by josesl16** [comment]
Reporting here on 22H2 with the new .2506 Update, same issue of crashing as soon as I change workspace. Toggling notification didn't work, but deleting the assembly cache did work for me fortunately.

**Comment by eggbean** [comment]
It must have been something in my settings, as I stopped the process in Task Manager and deleted the whole `hwtnb.net` directory and restarted it. I remade the most important key bindings (to move current window) and it seems to be working fine.

**Comment by w3b-beweb** [comment]
> > Have you tried the troubleshooting step mentioned in the release notes?
> 
> I didn't think of doing that as it was Windows that got updated and not SylphyHornPlusWin11. I deleted the two files I had in `assemblies` but it didn't help. It's still not working for me.
> 
> @w3b-beweb So it's working for you now?

Yes it works. Sorry for the delay in the response but, in the mean time, I managed to break my right ankle...

**Comment by eggbean** [comment]
Sorry to hear that. At least SylphyHornPlusWin11 is still working though.

EDIT: (reopening so that others get this information)
